DevTools/ContainerDebug/NanoDockerBuild.cmd

```diff
@@ -4,12 +4,10 @@ SETLOCAL EnableDelayedExpansion
 set LOCAL_BIN_STAGING_DIR=BinariesToCopy
 set VS_REMOTE_DEBUGGER_BIN="%VSINSTALLDIR%CoreCon\Binaries\Phone Tools\Debugger\target\x64"
 set VS_REMOTE_DEBUGGER_LIB="%VSINSTALLDIR%CoreCon\Binaries\Phone Tools\Debugger\target\lib"
-set VS_CRT_REDIST_REL[0]="%VcToolsRedistDir%onecore\x64\Microsoft.VC141.CRT"
-set VS_CRT_REDIST_REL[1]="%VcToolsRedistDir%onecore\x64\Microsoft.VC150.CRT"
-set VS_CRT_REDIST_DBG[0]="%VcToolsRedistDir%onecore\debug_nonredist\x64\Microsoft.VC141.DebugCRT"
-set VS_CRT_REDIST_DBG[1]="%VcToolsRedistDir%onecore\debug_nonredist\x64\Microsoft.VC150.DebugCRT"
+set VS_CRT_REDIST_REL="%VcToolsRedistDir%onecore\x64\Microsoft.VC142.CRT"
+set VS_CRT_REDIST_DBG="%VcToolsRedistDir%onecore\debug_nonredist\x64\Microsoft.VC142.DebugCRT"
```

Comment:
Would it make sense to add this as an additive option so we do not break older versions? I think the earlier PR did that, that's why there is a for loop.

Reply:
Sure - put 141, 150 and 142 (all three)?

Reply:
I think so, unless you have another opinion to consider.

Reply:
Done.